packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/HeadersInput/index.js

```diff
@@ -11,6 +12,10 @@ import { TextButton } from '@strapi/parts/TextButton';
 import { Field, FieldArray, useFormikContext } from 'formik';
 import { useIntl } from 'react-intl';
 
+const RemoveButtonContainer = styled(Box)`
```

Comment:
Using a row with alignItems flex-end does not work?

Reply:
unfortunately flex doesn't help here because the height of the row is based the height of the input which is label + input
<img width="466" alt="Capture d’écran 2021-10-06 à 08 59 48" src="https://user-images.githubusercontent.com/71838159/136155229-17804ad6-249d-482c-a89a-a0c86768960b.png">